src/scss/renderers.scss

```diff
@@ -46,12 +56,38 @@ $md-code-color: #fff;
   background-color: $md-code-color;
 }
 
+.md-inline-code {
+  word-wrap: break-word;
+  background-color: rgb(43, 43, 43);
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+}
+
 .md-img {
   max-width: 100%;
 }
 
 .md-link {
   color: common.$link-color;
-  text-decoration: underline;
-  text-underline-offset: 1rem;
+  text-decoration: none;
+  border-bottom: 0.1rem common.$link-color solid;
+}
+
+.md-blockquote {
+  margin-left: 2rem;
+  margin-right: 2rem;
+  margin-inline-start: 2rem;
+  margin-inline-end: 2rem;
+  border-left: 0.7rem solid #7f7f7f;
+  padding-left: 1rem;
+  color: #cccccc;
```

Comment:
Codacy found an issue: [Expected "#cccccc" to be "#ccc" (color-hex-length)](https://app.codacy.com/manual/sharmaabhyudaya/abhyudayasharma.github.io/pullRequest?prid=5731220)